### ISSUE TITLE
Bot doesn't discard wrong items

### ISSUE THREAD

**Issue by Lilipata** [kolton/d2bot-with-kolbot#2458]
Hi! I play manually, I config my character to pick up items. I prepared an archive with the items I want. My character doesn't discard the items I don't want after identify then. I don't know where's the problem.

Thanks!

**Comment by gtoilet** [comment]
um its manual so you do it your self it only picks up auto

**Comment by inherithandle** [comment]
make sure your inventory setting is correct in your char config.
```
        Config.Inventory[0] = [0,0,0,0,0,0,0,0,0,0];
	Config.Inventory[1] = [0,0,0,0,0,0,0,0,0,0];
	Config.Inventory[2] = [0,0,0,0,0,0,0,0,0,0];
	Config.Inventory[3] = [0,0,0,0,0,0,0,0,0,0];
```
if your inventory flags are all zeros, your bot won't discard picked item. assume you have a .nip file like:
```
[type] == armor # [defense] >= 3500
```
your bot will pick an armor and then see if the armor has the defense above 3000. but your bot won't sell this armor to NPC, if the armor has defense below 3500 and your inventory flags are all 0s.

**Comment by inherithandle** [comment]
do you mean you are on the `ManualPlayPick` mode? then your bot can't automatically discard picked item.

**Comment by gtoilet** [comment]
yes that is what i mean it will auto pick them while you manual play
if you want it to do everything set it and watch

**Comment by ARkaine2** [comment]
> um its manual so you do it your self it only picks up auto

Can you please tell how excatly to do automatic item picking with manual playing?

**Comment by gtoilet** [comment]
@ARkaine2  ,,,, was on front page not that far from this post


https://github.com/kolton/d2bot-with-kolbot/issues/2449#issuecomment-622989556
